### PR TITLE
Remove bundled ProximaNova font

### PR DESCRIPTION
**Pull request by sammacbeth**
- Removes ProximaNova font in CTL assets.
- Adds a mechanism for the platform to send asset URLs for custom fonts on `init`, allowing CTL fonts to be customized.
- We can use this in the extension to use bundled fonts that work in all locales (see https://github.com/duckduckgo/duckduckgo-privacy-extension/pull/1892/commits/3ca91a675d7cbb2d5b8a512135edcee2f1f3d327)
- For platforms that don't send asset URLs, they get system default fonts that match the privacy-dashboard.

### CODE REVIEW

File: src/features/click-to-load.js [hunk 1]
```diff
@@ -1615,6 +1616,10 @@ export default class ClickToLoad extends ContentFeature {
         const localizedConfig = getConfig(locale)
         config = localizedConfig.config
         sharedStrings = localizedConfig.sharedStrings
+        // update styles if asset config was sent
+        if (args?.assets) {
+            styles = getStyles(args.assets)
```
Comment (jonathanKingston) [inline src/features/click-to-load.js:1621]: Can we expose these on the class somehow? I'm trying to deprecate `args` lately.

File: src/features/click-to-load.js [hunk 2]
```diff
@@ -22,6 +22,7 @@ const titleID = 'DuckDuckGoPrivacyEssentialsCTLElementTitle'
 // @see {getConfig}
 let config = null
 let sharedStrings = null
+let styles = getStyles()
```
Comment (jonathanKingston) [inline src/features/click-to-load.js:25]: Do we ever use this before `init`?

Comment (sammacbeth) [reply src/features/click-to-load.js:25]: No, but this prevents a typescript error...

Comment (jonathanKingston) [reply src/features/click-to-load.js:25]: Can we change the type to make it optional? I think that reduces the risk of unexpected side effects

**Review by kzar — APPROVED** [review]

**Review by jonathanKingston — APPROVED** [review]
Thanks :)

**Pull request merged**